### ISSUE TITLE
Contact panel (left) & in chat name panel are now WHITE (not dark)

### ISSUE THREAD

**Issue by algotr8der** [m4heshd/whatsapp-desktop-dark#118]
Everything was working fine for months until today when my WhatsApp Desktop on my MacOS crashed. So I relaunched WhatsApp Desktop but it was WHITE so I ran WADark and I did not restore the backup for whatever reason. Just went through the options to run WADark for the first time. 

Once WhatsApp Desktop launched I noticed that the ONLY part that was dark was the background within a chat window (chat wallpaper)

The Contacts Panel (left) & the name panel within a chat are WHITE (not dark). 

This is bizarre because prior to this problem EVERYTHING was DARK.

Basically, either WADark stopped working altogether or its only darkening the chat wallpaper area.

My OS: 10.15.2 (19C57)
WhatsApp Version: Version ‪0.4.315‬

Please see screenshot - note I have blacked out the contact bar images and names for privacy reasons. 

As you can see from the screenshot the Contact Panel (left) and Name in the Chat panel are WHITE.

Please contact me if you have any questions. Thank you.

![Screen Shot 2020-01-23 at 11 14 24 PM](https://user-images.githubusercontent.com/1220850/73044193-79bff200-3e36-11ea-942f-2cedfd3ba3c7.png)

**Comment by renrique92** [comment]
Same to me, the problem it's already addressed, they're working on it

**Comment by Paft** [comment]
Hate to be that guy but how did you miss the other threads? There's like 5 excluding yours...

**Comment by Dr-Big-Man** [comment]
Same observation with me - but differently produced, so maybe details can be helpful.

As I couldn't start the downloaded version, I installed from source - once thru direct execution and once thru the app in `dist/macOS/WADark`. Both ways an error message was produced during `npm install` (log file mentioned in error message attached below):

```zsh
➜  $> npm install

> resourcehacker@4.2.51 postinstall /Users/drbigman/Desktop/whatsapp-desktop-dark-master/node_modules/resourcehacker
> node scripts/install.js

Downloading ResourceHacker by Angus Johnson...
_http_client.js:141
  let expectedProtocol = defaultAgent.protocol;
                                      ^

TypeError: Cannot read property 'protocol' of undefined
    at new ClientRequest (_http_client.js:141:39)
    at request (http.js:46:10)
    at Object.get (http.js:50:15)
    at Object.<anonymous> (/Users/drbigman/Desktop/whatsapp-desktop-dark-master/node_modules/resourcehacker/scripts/install.js:23:20)
    at Module._compile (internal/modules/cjs/loader.js:1151:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1171:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! resourcehacker@4.2.51 postinstall: `node scripts/install.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the resourcehacker@4.2.51 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/drbigman/.npm/_logs/2020-01-25T08_08_01_286Z-debug.log

➜  $> node .

~~~~ WhatsApp Desktop Dark Mode by m4heshd ~~~~


Checking for dark mode updates.. (Current version - v0.3.4941)

This process takes at least a minute to complete. So please be patient and let it do the magic..

Wait for the application to kill WhatsApp and start the process..

"override.json" was found. Customizing styles..

Theme "Default" was successfully applied.

TIP: You can create/download custom themes using "override.json" (Instructions are in the documentation)

Backing up..
Extracting..
Injecting styles..
Replacing files..
Cleaning up..

All done. May your beautiful eyes burn no more.. Enjoy WhatsApp Dark mode!! :)

Press any key to exit..
➜  $>
```
[2020-01-25T08_08_01_286Z-debug.log](https://github.com/m4heshd/whatsapp-desktop-dark/files/4111754/2020-01-25T08_08_01_286Z-debug.log)

P.S.: Just noticed that the log file from previous execution of `npm install` is much larger - so attaching it here 
[2020-01-25T07_55_26_086Z-debug.log](https://github.com/m4heshd/whatsapp-desktop-dark/files/4111768/2020-01-25T07_55_26_086Z-debug.log)
 in case that it contains further hints what went wrong?

**Comment by Redi1973** [comment]
I have the same problem as my predecessors.
The Dark Mode could be installed without any problems, but on the left side where my contacts are listed, the Messenger unfortunately remains in white.
Would be grateful for your help in getting it into full dark mode.
WhatsApp Version 0.4.315

**Comment by algotr8der** [comment]
Isn't is possible to revert to an older version of WADark?

**Comment by Redi1973** [comment]
With whatsapp version 0.3.9309 the Dark Mode is fully functional.
Everything in black as it should be

**Comment by mrmelon54** [comment]
![main whatsapp screenshot](https://user-images.githubusercontent.com/48031324/73403065-918bf000-42e6-11ea-9b3c-a811bb5f971c.png)
![emoji selector](https://user-images.githubusercontent.com/48031324/73403105-a8cadd80-42e6-11ea-86e3-e1cd1e1a8941.png)
![version screen](https://user-images.githubusercontent.com/48031324/73403171-c9933300-42e6-11ea-9485-a2773fc69170.png)
Here's some screenshots of version 0.4.315‬ so you can see what is broken

**Comment by algotr8der** [comment]
Whatsapp Desktop FORCED me to update/download the latest updated version
TODAY (Version ‪0.4.315‬).

After Install it works fine (all white ofcourse) BUT what happens after I
run WADark, and then when it relaunches Whatsapp after injecting styles/etc
Whatsapp throws up an exception error message indicating that *Couldnt
Update Automatically* (see screenshot below).

So basically, we already know the latest WADark was broken (not darkening)
and TODAY it is entirely incompatible with Whatsapp Desktop as it seems
Whatsapp Desktop is performing some kind of verification that causes it to
realize it is NOT the default and throws up an error.

Can anyone confirm my findings?

[image: Screen Shot 2020-01-30 at 8.02.42 AM.png]
[image: Screen Shot 2020-01-30 at 8.00.47 AM.png]

**Comment by algotr8der** [comment]
I forgot to mention I have MacOS version 10.15.2 (19C57) - Catalina.

**Comment by vibonacci** [comment]
Facebook destroyed the mod with the latest version unfortunately. Hopefully we get a patch soon... my eyes and sleep are suffering.

**Comment by Redi1973** [comment]
My place looks exactly like yours, algotr8der
I can soften it a bit by setting the chat background to black, but on the left side of my contacts everything stays white.
I have Windows 10 64 bit Insinder Build 19551.1005

**Comment by abicelis** [comment]
Can verify @algotr8der 's findings. 
Same thing happens on macOS Catalina 10.15.2
Whatsapp version 0.4.315
See pic:

<img width="615" alt="Screen Shot 2020-01-30 at 3 51 01 PM" src="https://user-images.githubusercontent.com/9850854/73489191-399dc980-4378-11ea-9b8b-9483a3141621.png">

**Comment by m4heshd** [comment]
Closing this since the issue is a duplicate of #113 and has been fixed in the latest release.

**Comment by algotr8der** [comment]
Seems like this same issue is back again. The right panel is white.
See attached.

Whatsapp Desktop version is 2.2017.6

**Comment by algotr8der** [comment]
My OS is MacOS 10.15.2 Catalina
